web/packages/shared/components/AccessRequests/NewRequest/RequestCheckout/RequestCheckout.tsx

```diff
@@ -623,6 +627,29 @@ function TextBox({
   );
 }
 
+function getPrettyResourceKind(kind: ResourceKind): string {
+  switch (kind) {
+    case 'role':
+      return 'Role';
+    case 'app':
+      return 'Application';
+    case 'node':
+      return 'Server';
+    case 'resource':
+      return 'Resource';
+    case 'db':
+      return 'Database';
+    case 'kube_cluster':
+      return 'Kubernetes';
```

Comment:
```suggestion
      return 'Kube cluster';
```

`tsh kube ls` uses this as a column name as well.

Reply:
I decided to use `Kubernetes` because in the unified resources we use this name, so I thought that it would be consistent: 
<img width="389" alt="image" src="https://github.com/gravitational/teleport/assets/20583051/421bc04c-2016-4038-a3cc-06336302233e">
WDYT?

Reply:
Oh, in that case I guess it's okay. I didn't realize that there are other places which use plain "Kubernetes" already.